src/asammdf/blocks/mdf_v4.py

```diff
@@ -5693,15 +5693,13 @@ def extend(self, index: int, signals: list[tuple[NDArray[Any], NDArray[Any] | No
                             if size % 2:
                                 size += 1
                                 elem = elem + b"\0"
-                            data.append(UINT32_p(size))
-                            data.append(elem)
+                            data.extend((UINT32_p(size), elem))
                             off += size + 4
                     else:
                         for elem in signal:
                             offsets.append(off)
                             size = len(elem)
-                            data.append(UINT32_p(size))
-                            data.append(elem)
+                            data.extend((UINT32_p(size), elem))
```

Comment:
Is this automatically done by ruff?

Reply:
Not really, this actually not part of the pylint rules, instead i ran some of the preview rules: https://docs.astral.sh/ruff/rules/repeated-append/
Even then this is only done automatically if you call ruff with `--unsafe-fixes`. 
Should i revert these?

Reply:
It's ok I guess, it was more a curiosity